site/src/sphinx/server-annotated-service.rst

```diff
@@ -1232,3 +1231,54 @@ Then, you can annotate your service method with your annotation as follows.
         @MyProducibleType  // the same as @Produces("application/xml")
         public MyResponse hello(MyRequest myRequest) { ... }
     }
+
+
+Annotation-based HTTP Response headers/trailers
```

Comment:
How about: `Specifying additional response headers`? I think we don't need to mention 'annotation-based' because this page is all about annotation-based services.